numba_cuda/numba/cuda/tests/cudapy/test_warp_ops.py

```diff
@@ -144,6 +146,33 @@ def test_shfl_sync_xor(self):
         compiled[1, nelem](ary, xor)
         self.assertTrue(np.all(ary == exp))
 
+    def test_shfl_sync_const_mode_val(self):
```

Comment:
We should also have a variant of this test that uses `compile_ptx()` with `cc=(10, 0)` to ensure that all is well when going through the path that uses LLVM 18.x IR.

Reply:
[b359f9a](https://github.com/NVIDIA/numba-cuda/pull/231/commits/b359f9a78cba7d8ef00512920f56d5d74f0343a6)